apps/backoffice-v2/src/common/hooks/useZodSearchParams/utils/deep-string-to-null.ts

```diff
@@ -0,0 +1,15 @@
+export const deepStringToNull = (value: unknown) => {
+  if (typeof value === 'string' && value === 'null') return null;
```

Comment:
First condition is redundant.
```suggestion
  if (value === 'null') return null;
```